x-pack/plugins/fleet/public/components/agent_enrollment_flyout/download_instructions.tsx

```diff
@@ -5,48 +5,36 @@
  * 2.0.
```

Comment:
`DownloadStep` which returns the `DownloadInstructions` component in now never used. It was used in `ManagedSteps` in case of Kubernetes only but I remove this as the functionality of the step is merged in the `InstallManagedAgentStep`.
Also it is never used by fleet server installation. Maybe it was used in the past.

So I suggest to delete the step and rename the `DownloadInstructions` (only used by kubernetes case) to `KubernetesInstructions`.
@criamico WDYT ?